### PR TITLE
[Storage] Upgrade SRP SDK to 9.0.0 and fix some issue

### PR DESCRIPTION
**Pull request by blueww**
## Description

This PR targets 12/4 release.

1. Upgrade SDK to 9.0.0 per issue https://github.com/Azure/azure-powershell/issues/7780
2. (breaking) get global location is remove since SRP SDK upgrade, so -Location parameter become mandatory in Get-AzureRMStorageUsage
3. (breaking) Change default parameter set to Oauth parameter set as the next release allows breaking https://github.com/Azure/azure-powershell/issues/7143
4. Fix issue https://github.com/Azure/azure-powershell/issues/7805, but set Account name to "[AccessToken]" when Context is an Oauth Context (aligned with SAS context)

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by blueww** [comment]
@maddieclayton , @markcowl 
I have rebase this branch on preview branch, and resolve the conflicts and removed the breaking change warning, as we discussed in the meeting.
Please help to review.

File: src/ResourceManager/Storage/Commands.Management.Storage/StorageAccount/GetAzureStorageUsage.cs [hunk 1]
```diff
@@ -22,12 +22,11 @@
 
 namespace Microsoft.Azure.Commands.Management.Storage.StorageAccount
 {
-    [GenericBreakingChange("Parameter 'Location' will change from optional to mandatory in a future release.")]
     [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "StorageUsage"), OutputType(typeof(PSUsage))]
     public class GetAzureStorageUsageCommand : StorageAccountBaseCmdlet
     {
         [Parameter(
-            Mandatory = false,
+            Mandatory = true,
```
Comment (maddieclayton) [inline src/ResourceManager/Storage/Commands.Management.Storage/StorageAccount/GetAzureStorageUsage.cs]: Please update the help files for the two cmdlets affected by this PR.

Comment (blueww) [reply src/ResourceManager/Storage/Commands.Management.Storage/StorageAccount/GetAzureStorageUsage.cs]: Thanks for the catch! Help is updated.

**Comment by maddieclayton** [comment]
This will be ready whenever we are taking changes for the breaking change release.

**Comment by cormacpayne** [comment]
@blueww I re-targted your PR to our new default branch, `master`, so you will need to pull from that branch in order to resolve the merge conflicts.

_Note_: since the next release is the GA of the `Az` module, we have removed all reference docs to `AzureRM` cmdlets, so you will need to regenerate your help based on the `Az` cmdlets.

**Comment by blueww** [comment]
@cormacpayne 

I have rebased from master branch, and re-generated help file.
Please help to check.

**Comment by blueww** [comment]
@cormacpayne 
As Az 0.7.0 has released, would you please help to merge the PR?
Thanks!

**Pull request merged**